### PR TITLE
build: generate api.json for base package

### PR DESCRIPTION
**Pull request by ilhan007**
Let's start generating api.json for the base package as well - there are classes, especially data types used within components, for example (valueState, type: ValueState). This way, the API picture will be more complete.

### CODE REVIEW

**Comment by ilhan007** [comment]
FYI @g-cheishvili

**Comment by vladitasev** [comment]
@dobrinyonkov FYI for storybook integration

File: packages/base/src/types/Float.js [hunk 1]
```diff
@@ -1,5 +1,16 @@
 import DataType from "./DataType.js";
 
+/**
+ * @class
+ * Float data type.
+ *
+ * @constructor
+ * @extends DataType
+ * @author SAP SE
+ * @alias sap.ui.webcomponents.base.types.Integer
```
Comment (vladitasev) [inline packages/base/src/types/Float.js:10]: Integer -> Float

Comment (ilhan007) [reply packages/base/src/types/Float.js]: Thanks, done

**Pull request merged**